### ISSUE TITLE
setOnlyAlertOnce for Android Notifications

### ISSUE THREAD

**Issue by Ameggg** [florent37/Flutter-AssetsAudioPlayer#316]
Could you add the ability to decide whether or **_not to repeatedly alert_** the user that the **notification metas** have been updated on Android.

Useful StackOverflow Post: [https://stackoverflow.com/a/15538209/13835434](url)

### I do not want the notification to appear every time the metas are updated:
![example](https://user-images.githubusercontent.com/41295977/88983347-cd2a6580-d2db-11ea-93fe-1342f6e03416.png)

**Comment by Ameggg** [comment]
I was able to modify the plugin locally and achieved the desired feature:

Edit: **android/src/main/kotlin/com/github/florent37/assets_audio_player/notification/NotificationService.kt**
Line 288: added **.setOnlyAlertOnce(true)** after **.setContentText(action.audioMetas.artist)**

I'm still not an expert at PR and linking Flutter to Native code :)

**Comment by elmissouri16** [comment]
yes will great to have a solution for it its work fine on some devices others each time you switch a song its popup again
